status-im/status-react
# Feature/discover dapps details 1973

fixes #1973 but also includes #2042 because the app could not be navigated to the dapp screen without it.

### Summary:
DApp details screen which is shown when a dapp is tapped on either in the DApp carousel (#1968) or All Dapps (#1972)  screen under Discover. User can see the details about the dapp on this screen or Open it using the main action.

### Steps to test:

Scenario 1:

- Open Status
- Go to Discover
- Tap on any DApp in the carousel
- See the dapp details and tap Open to interact with the dapp

Scenario 2:
- Open Status
- Go to Discover
- Tap on "all" action above the dapps carousel
- When on All DApps screen, tap on any dapp
- See the dapp details and tap Open to interact with the dapp

Features omitted from the scope of this PR:
- Fine styling 
- dapps carousel is wip
- Description in dapp details intentionally missing (we don't have that data yet)

[comment]: # (PRs will only be accepted if squashed into single commit.)
status: ready
